product_docs/docs/pem/8/registering_agent.mdx

```diff
@@ -97,43 +97,84 @@ For information about using the pemworker utility to register a server, see [Reg
 
     ```shell
-    mkdir /home/<edb>/.pem
-    mkdir /home/<edb>/.pem/logs
-    chmod 700 /home/<edb>/.pem
-    chmod 700 /home/<edb>/.pem/logs
+    $ mkdir /home/edb/pem
+    $ mkdir /home/edb/pem/logs
+    $ chmod 700 /home/edb/pem
+    $ chmod 700 /home/edb/pem/logs
     ```
 
 2.  Register the agent with PEM server:
 
     ```shell
-    ./pemworker --register-agent --pem-server <172.19.11.230> --pem-user <postgres> --pem-port <5432> --display-name <non_root> --cert-path /home/<edb> --config-dir /home/<edb>
+    $ export PEM_SERVER_PASSWORD=edb
+    $ /usr/edb/pem/bin/pemworker --register-agent --pem-server <172.19.11.230> --pem-user postgres --pem-port 5432 --display-name non_root_pem_agent --cert-path /home/edb/pem --config-dir /home/edb/pem
 
-    The above command creates agent certificates and an agent configuration file (``agent.cfg``) in the ``/home/edb/.pem`` directory. Use the following command to assign read and write permissions to these files:
+    # The above command creates agent certificates and an agent 
+    # configuration file (`agent.cfg`) in the `/home/edb/pem` directory. 
+    # Use the following command to assign read and write permissions to 
+    # these files:
 
-    ``chmod -R 600 /home/edb/.pem/agent*``
+    $ chmod -R 600 /home/edb/pem/agent*
     ```
 
 3.  Change the parameters of the `agent.cfg` file:
 
     ```ini
-    agent_ssl_key=/home/edb/.pem/agent<id>.key
-    agent_ssl_crt=/home/edb/.pem/agent<id>.crt
-    log_location=/home/edb/.pem/worker.log
-    agent_log_location=/home/edb/.pem/agent.log
+    $ vi /home/edb/pem/agent.cfg
+    agent_ssl_key=/home/edb/pem/agent<id>.key
+    agent_ssl_crt=/home/edb/pem/agent<id>.crt
+    log_location=/home/edb/pem/worker.log
+    agent_log_location=/home/edb/pem/agent.log
     ```
 
-4.  Update the values for the configuration file path and the user in the `pemagent` service file:
+    Where `<id>` is the assigned PEM agent id.
+
+4.  Create a `tmp` directory, set the environment variable and start the agent:
+    ```ini
+    $ mkdir /home/edb/pem/tmp
+    
+    # Create a script file, add the environment variable, give permissions and execute:
+    $ vi /home/edb/pem/run_pemagent.sh
+    #!/bin/bash
+    export TEMP=/home/edb/agent/tmp
+    /usr/edb/pem/agent/bin/pemagent -c /home/edb/agent/etc/agent.cfg
+    $ chmod a+x /home/edb/pem/run_pemagent.sh
+    $ cd /home/edb/pem
+    $ ./run_pemagent.sh
+    ```
+
+    Your PEM agent is registered and started with `edb` user at this point. If the machine restarts then this agent do not restart automatically and need to start manually using the above command.
+
+    Optionally, you can create the service for this PEM agent as root user to start this agent automatically at machine restart as follows:
+
+5.  Update the values for the configuration file path and the user in the `pemagent` service file as superuser:
```

Comment:
Steps 5 and 6 are only necessary if you choose to set up a service to restart the agent automatically, correct? If this is true we should nest these steps making them step 4a and 4b.